.circleci/config.yml

```diff
@@ -27,10 +27,11 @@ executors:
           DATABASE_URL: 'postgres://ubuntu@localhost:5432/coursemology_test'
           COLLECT_COVERAGE: << parameters.collects_rails_coverage >>
 
-      - image: cimg/postgres:16.1
+      - image: pgvector/pgvector:pg16
```

Comment:
squash 6b52688f966ae1321cc44c18746b6de68104ecd6 with first commit c2c599e1199d33e16e22f25ec9c008080b43c21c where you added `pgvector` gem

Reply:
Squashed